src/judging_round_display.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import renderer from 'react-test-renderer';
+import {mount, shallow} from 'enzyme';
+import JudgingRoundDisplay from './judging_round_display';
+
+
+describe('judging round display with no judging round', () => {
+    it('says No Judging Round Selected', () => {
+	const display = shallow(<JudgingRoundDisplay judging_round={null}/>);
+	expect(display.text()).toContain('No Judging Round Selected');
+    });
+});
+
+
+describe('judging round display with judging round', () => {
+    beforeEach(() => {
+	global.fetch = jest.fn().mockImplementation(
```

Comment:
This works well. We've also been trying out https://github.com/jefflau/jest-fetch-mock which has some nice test helpers for setting up these promises.

Reply:
Not going to dig into that now, but will keep it in mind for later.